calico-cloud_versioned_docs/version-3.15/get-started/connect/cc-arch-diagram.mdx

```diff
@@ -0,0 +1,48 @@
+| Fluentd                        | Open-source data collector for unified logging. Collects and forwards {{prodname}} logs (flows, DNS, L7) to log storage. | &bull;  TCP 8080 to Guardian<br />&bull;  TCP 9080 from Prometheus API service |
+| Guardian                       | An agent running in each managed cluster that proxies communication between the {{prodname}} tunnel server and your managed cluster. Secured using TLS tunnels.<br /> | &bull; Port 9000 to tunnel server<br />&bull; TCP 6443 to Kubernetes API server<br />&bull; TCP 6443 from {{prodname}} components |
+| Installation endpoints         | Endpoints at  `*.calicocloud.io` and `*.projectcalico.org`.  | TCP 443 for both                                             |
+| Intrusion detection controller | Handles integrations with threat intelligence feeds and {{prodname}} custom alerts. | &bull; TCP 8080 to Guardian<br />&bull; TCP 6443 to Kubernetes API server |
+| Image Assurance                | Identifies vulnerabilities in container images that you deploy to Kubernetes clusters.  Components of interest are: <br /><br />**Admission controller**<br />Uses Kubernetes Validating Webhook Configuration https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/ to control which images can be used to create pods based on scan results. <br /><br />**API**<br />Isolates tenant data and authorizes all external access to Image Assurance data. **Note:** {{prodname}} does not store registry credentials in its database and does not pull customer images into the {{prodname}} control plane.<br /> | &bull; TCP 8080 to Guardian<br /> &bull;  TCP 6443 to Kubernetes API server |
```

Comment:
Do you really want the full link text in the doc?

Reply:
No, good catch!